gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/EmailServiceImpl.java

```diff
@@ -137,14 +138,13 @@ public Maybe<Email> send(Domain domain, Application client, io.gravitee.am.model
 
     private Maybe<Email> prepareAndSend(Domain domain, Application client, io.gravitee.am.model.Template template, User user) {
         // get raw email template
-        return getEmailTemplate(template, user).map(emailTemplate -> {
+        return getEmailTemplate(template, user).flatMap(emailTemplate -> {
```

Comment:
Question: Why are you switching to flatMap and so Initialize a Maybe whereas the previous implementation was OK IMO.

Reply:
I implemented something there but removed it after. I'll rollback to thoses changes